src/components/Modal/readme.md

```diff
@@ -14,13 +14,15 @@ class EmptyModal extends React.Component {
         };
         this.handleOnClick = this.handleOnClick.bind(this);
         this.handleOnClose = this.handleOnClose.bind(this);
+        this.buttonRef = React.createRef();
```

Comment:
We don't want to make this always, instead we should find a solution that works out of the box each time we use the modal